### PR TITLE
Handle the existence or not of the `preflight_` prefix in rego

### PR DESCRIPTION
**Pull request by j-fuentes**
This allows the `preflight_` to exist or not if the schema version of the package is lower than "0.1.0". It assumes the artificial prefix does not exist if the schema version is higher than "0.1.0".

Related: #27

### CODE REVIEW

**Comment by charlieegan3** [comment]
/approve 

Can lgtm too but was going to let @wwwil have a look.

File: pkg/exporter/exporter.go [hunk 1]
```diff
@@ -32,9 +32,13 @@ const (
 )
 
 func ruleToResult(ruleID string) string {
-	return fmt.Sprintf("preflight_%s", strings.ReplaceAll(ruleID, ".", "_"))
+	return strings.ReplaceAll(ruleID, ".", "_")
```
Comment (wwwil) [inline pkg/exporter/exporter.go]: Isn't this unnecessary now as rule IDs are required to be valid Rego names?

Comment (j-fuentes) [reply pkg/exporter/exporter.go]: They are not strictly required though.

We check that in the linter, which flags recommendations but, at least for now, we don't impose those rules.

If at some point we want to impose them, we would run those checks at the beginning of the execution of Preflight.

Comment (wwwil) [reply pkg/exporter/exporter.go]: Yeah, I feel that we should be enforcing it. I think packages should be linted when they are loaded as part of a `preflight check` and packages that fail should be ignored. This PR is probably not the place to make that change though.

**Pull request merged**